files/image_config/hostcfgd/hostcfgd

```diff
@@ -304,13 +304,33 @@ class HostConfigDaemon:
             add = False
 
         self.iptables.iptables_handler(key, data, add)
+        
+    def feature_status_handler(self, key, data):
+        status_data = self.config_db.get_table('FEATURES')
```

Comment:
Naming convention for tables in SONiC DB is singular form. Can you change this to `FEATURE` throughout?

Reply:
update in all files